README.md

```diff
@@ -1,42 +1,124 @@
 
-## Documentation
-
-For detailed documentation about GenSX, please refer check out the [docs](https://gensx.dev).
+Check out the [documentation](https://gensx.dev/basic-concepts) to learn more about building LLM applications with GenSX.
 
 ## Getting Started
 
-1. Install dependencies:
+To create a new GenSX project, run the following command:
 
 ```bash
-pnpm install
+npm create gensx@latest my-app
 ```
 
-2. Build all packages:
+To add GenSX to an existing project, run the following command and follow the instructions described [here](https://www.npmjs.com/package/gensx):
 
 ```bash
-pnpm build:all
+npm install gensx @gensx/openai
+```
+
+Check out the [Quickstart Guide](https://gensx.dev/quickstart) for more details on getting started.
+
+## Examples
+
+This repo contains a number of [examples](./examples) to help you get up and running with GenSX.
+
+## Building a workflow, the declarative way!
+
+Most LLM frameworks are graph oriented--you express your workflow with nodes, edges, and a global state object. GenSX takes a different approach--you compose your workflow with components, and GenSX handles the execution for you.
+
+You start by defining your components:
+
+```tsx
+import { gsx } from "gensx";
+import { OpenAIProvider, ChatCompletion } from "@gensx/openai";
+
+// Define the input props and output type for type safety
+interface CreateOutlineProps {
+  prompt: string;
+}
+type CreateOutlineOutput = string;
+
+// Create a reusable component that can be composed with others
+const CreateOutline = gsx.Component<CreateOutlineProps, CreateOutlineOutput>(
+  "CreateOutline",
+  async ({ prompt }) => {
+    return (
+      <ChatCompletion
+        model="gpt-4o-mini"
+        messages={[
+          {
+            role: "user",
+            content: `Create an outline for an article about ${prompt}`,
+          },
+        ]}
+      />
+    );
+  },
+);
+
+interface WriteArticleProps {
+  outline: string;
+}
+type WriteArticleOutput = string;
+
+const WriteArticle = gsx.Component<WriteArticleProps, WriteArticleOutput>(
+  "WriteArticle",
+  async ({ outline }) => {
+    return (
+      <ChatCompletion
+        model="gpt-4o-mini"
+        messages={[
+          {
+            role: "user",
+            content: `Use this outline to write a detailed article: ${outline}`,
+          },
+        ]}
+      />
+    );
+  },
+);
 ```
 
-## Run an example
+Then you can compose your components together to create a workflow:
 
-```bash
-OPENAI_API_KEY=<my api key> turbo run run --filter="./examples/blogWriter"
+```tsx
+// Execute the workflow with the OpenAI provider
+const result = await gsx.execute<string>(
+  <OpenAIProvider apiKey={process.env.OPENAI_API_KEY}>
+    <CreateOutline prompt="the future of LLM dev tools">
+      {(outline) => <WriteArticle outline={outline} />}
+    </CreateOutline>
+  </OpenAIProvider>,
+);
+
+console.log(result);
 ```
 
+## Working with this repo
```

Comment:
Before this section, should we add an `## Examples` section that replicates the table in the examples readme?

Reply:
sounds reasonable. will be a good way to beef up the current example section (which was clearly so small that it's easy to miss 😁 )